R/summariseRecordCount.R

```diff
@@ -11,108 +11,117 @@
 #' @export
 #'
 summariseRecordCount <- function(omopTable, unit = "year", unitInterval = 1, ageGroup = NULL) {
```

Comment:
why is all this commented?

Reply:
Because I was working on it and I hadn't finished at the time of doing the pull request

Reply:
(i wanted to do another pull request for summariseRecordCount)